### ISSUE TITLE
Discussion

### ISSUE THREAD

**Issue by arif98741** [jacurtis/squish#1]
Sir,
I am a continuous follower of you and always do that. Can you share more tutorials on Laravel advanced,Squash, API or others technology that you knows? People like you are spreading knowledge all over the world. God bless You so much

**Comment by jacurtis** [comment]
Yeah thanks for following. I am getting back into things and making more tutorials now. I should have a lot more coming out this year. I hope to get back onto a normal schedule again this year with more tutorials.

I have a video that is already uploaded to YouTube actually and will be released tomorrow. It is about how to run your own websocket server without needing to use a service like Pusher. It is really good. it is already complete and you should see it on YouTube tomorrow.

**Comment by arif98741** [comment]
Nice to hear from you sir. I started my journey with laravel by You. Hope that, I will learn more from you
